{{cookiecutter.project_slug}}/_/development/php/docker-compose.yml

```diff
@@ -20,9 +20,9 @@ services:
     image: postgres
     volumes:
       - database_data:/var/lib/postgresql/data
-{%- elif cookiecutter.database == 'MySQL/MariaDB' %}
+{%- elif cookiecutter.database == 'MySQL' %}
   database:
-    image: mariadb
+    image: MySQL
```

Comment:
Argh, that was a near miss. Too much search + replace. Needs to be reverted. :grimacing: